woocommerce/woocommerce
# add: easyship to shipping task for AU, NZ, SG, HK

- easyship added to AU in double layout
- easyship added to SG, HK and NZ in single layout
- removed skydropx from MX, CO

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/team-ghidorah/issues/272. Please check that issue for the P2 that has the updates.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Set your store country as one of SG, HK, AU, NZ, MX, CO 
1. Turn off marketplace suggestions in woo settings > advanced
1. For SG, HK, NZ, you should see the shipping task in the task list, and going to it should show you easyship as a single partner
For AU, you should see the shipping task in the task list, and going to it should show you easyship as a double partner alongside shipstation
1. For MX, CO you should not see a shipping task

To reset the status of the tasklist between testing different countries, please go to woocommerce settings > shipping and delete all the entries. And also use WooCommerce Beta Tester plugin and use the WCA Test Helper's options UI to search for and delete all options with "shipping" in them.

![image](https://github.com/woocommerce/woocommerce/assets/27843274/13d4c540-fbae-4872-b87c-eddc1acb2e36)

![image](https://github.com/woocommerce/woocommerce/assets/27843274/b5095c3a-ac2f-45a0-be49-02b35c4bc7a6)




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
